### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by minad** [emacs-exwm/exwm#41]
@Stebalien How do you feel about tagging a new release of Xelb and EXWM soon? I've switched from Emacs 29 to 30 about a week ago, and overall EXWM feels quite stable to me on both Emacs versions in its current state. After a release we could focus on larger restructurings, e.g., related to tab-bar or keymap handling.

- [x] Update https://github.com/emacs-exwm/exwm/wiki#changelog
- [x] Tag Xelb 0.19
- [x] Create Xelb GH release
- [x] EXWM: Update Xelb dependency to 0.19
- [x] Tag EXWM 0.29
- [x] Create EXWM GH release

**Comment by Stebalien** [comment]
Honestly, that sounds like a great idea. I can help out if you're willing to wait for the weekend, otherwise, go for it.

**Comment by minad** [comment]
Okay, I have a bit of time right now. I'll move forward with the release.

**Comment by minad** [comment]
I just had to bump EXWM again, now to version 0.30. Unfortunately the ELPA infrastructure doesn't allow us to bump the version of Xelb and EXWM at the same time. We should keep this in mind for the next time.

~~~
======== Building tarball archive/exwm-0.29.tar...
Build error for archive/exwm-0.29.tar: (error "Required xelb-0.19 > than available (0.18.0.20240415.160551)")
######## Build of package archive/exwm-0.29.tar FAILED!!
~~~
